### DISCUSSION TITLE
Height miscalculation

### DISCUSSION THREAD

**Discussion by abrudz** [community/community#9938] [Other Feature Feedback, Questions, & Ideas] [answered]
The body is too tall for the window:
![image](https://user-images.githubusercontent.com/10961427/148802168-bc477d98-64ca-4e89-bff3-7796ac3d4490.png)

**Comment by colinwm** [comment]
Nice find! This is on Firefox, right? I'll look into it

**Reply by abrudz** [reply]
Yes, Firefox 95.0.2

**Comment by rockwotj** [comment]
Yeah hitting this on Firefox as well

**Comment by colinwm** [comment, marked as answer]
This should be fixed now! Thanks for reporting!
